docs/infrastructure/create-environment.md

```diff
@@ -27,7 +31,10 @@ This is the initial manual process to create a new environment like review, dev,
 
 ## Infra secrets
 
-- Add the infrastructure secrets to the _inf_ key vault `kv-manbrs-[environment]-inf`
+Add the infrastructure secrets to the _inf_ key vault `kv-manbrs-[environment]-inf`:
+
+- For entra ID authentication (when `enable_auth` is true): aad-client-audiences, aad-client-id, aad-client-secret
```

Comment:
I've opened https://github.com/NHSDigital/dtos-manage-breast-screening/pull/495 to rename this to `enable_entra_id_authentication` so if we agree with that change we'll need to update here.